jest.config.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  preset: 'react-native',
+  preset: 'jest-expo',
```

Comment:
```suggestion
  preset: 'react-native',
```

let's not mix vanilla RN project with expo